src/vuln_analysis/utils/functions_parsers/lang_functions_parsers.py

```diff
@@ -126,5 +135,12 @@ def is_same_package(package_name_from_input, package_name_from_tree):
     def get_constructor_method_name():
         return None
 
-    def get_class_name_from_class_function(self):
+    @staticmethod
+    def get_class_name_from_class_function(func: Document):
+        pass
+
+    def is_package_imported(self, code_content: str, identifier: str, callee_package: str = "") -> bool:
+        pass
+
+    def extract_method_name_with_params(self, java_src: str) -> str | None:
```

Comment:
@tmihalac Parameter name should be independent of ecosystem
```suggestion
    def extract_method_name_with_params(self, src: str) -> str | None:
```